scrapy/utils/gz.py

```diff
@@ -51,8 +51,12 @@ def gunzip(data):
     return output
 
 _is_gzipped_re = re.compile(br'^application/(x-)?gzip\b', re.I)
+_is_octetstream_re = re.compile(br'^(application|binary)/octet-stream\b', re.I)
```

Comment:
Nice trick. It would make the final boolean expression a tiny bit easier to read.

```python
return (_is_gzipped(ctype) is not None or
            (_is_octetstream(ctype) is not None and
             cenc in (b'gzip', b'x-gzip')))
```
:+1: